SolARSample_SLAM_Multi/main.cpp

```diff
@@ -244,7 +273,25 @@ int main(int argc, char **argv) {
 				framePoses.push_back(keyframe2->getPose());
 				LOG_INFO("Number of initial point cloud: {}", pointCloudManager->getNbPoints());
 				LOG_INFO("Number of initial keyframes: {}", keyframesManager->getNbKeyframes());
+#ifdef SEMANTIC_ID
```

Comment:
nit: you don't have to, but can reduce preprocessor directive, I find it clearer
```
bootstrapOk = true;
#ifdef SEMANTIC_ID
if ( !fnSegment(boost::static_pointer_cast<Frame>(keyframes[0]))) {
  bootstrapOk = false;
} else {
 ... // same without bootstrapOk = true;
}
#endif
```